Extension/src/LanguageServer/copilotCompletionContextProvider.ts

```diff
@@ -133,55 +135,43 @@ export class CopilotCompletionContextProvider implements ContextResolver<Support
             telemetry.addRequestMetadata(documentUri, caretOffset, context.completionId,
                 context.documentContext.languageId, { featureFlag: snippetsFeatureFlag });
             const docUri = vscode.Uri.parse(documentUri);
+            const getClientForTime = performance.now();
             const client = clients.getClientFor(docUri);
+            const getClientForDuration = CopilotCompletionContextProvider.getRoundedDuration(getClientForTime);
+            logMessage += ` (getClientFor elapsed:${getClientForDuration}ms) `;

```

Comment:
I'm seeing the ` ` space at the end lead to a double space.

Reply:
![image](https://github.com/user-attachments/assets/b575ed52-cae5-4e40-9e00-d1dd55fe8451)

Reply:
@sean-mcmanus thank, I reworked the output now, let me know!